src/test/java/org/fundacionjala/coding/kevinherrera/crypto/TestCesarCrypto.java

```diff
@@ -0,0 +1,72 @@
+package org.fundacionjala.coding.kevinherrera.crypto;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * test CesarCrypto class.
+ */
+public class TestCesarCrypto {
+    private static final int ENCODE_KEY_3 = 3;
+    private static final int ENCODE_KEY_MINUS_3 = -3;
+    private static final int ENCODE_KEY_5 = 5;
+    private static final int ENCODE_KEY_MINUS_5 = -5;
+    private static final String ENCODE_KEY_STRING = "LIMON";
+
+
+    /**
+     * test encode.
+     */
+    @Test
+    public void testEncodeNormal() {
+        NormalCrypto crypto = new NormalCrypto();

```

Comment:
You are repeating this line of code in all your tests
Please move it to a @before method.